cli/command/engine/check.go

```diff
@@ -28,8 +28,8 @@ func newCheckForUpdatesCommand(dockerCli command.Cli) *cobra.Command {
 	var options checkOptions
 
 	cmd := &cobra.Command{
-		Use:   "check [OPTIONS]",
-		Short: "Check for available engine updates",
+		Use:   "check-for-upgrades [OPTIONS]",
```

Comment:
Not a fan of naming it `check-for-upgrades`. Perhaps `list`, `browse`, `list-updates`? t.b.h. (but separate discussion), I think there's too many options on this command for filtering "types" of updates.

Wondering if we should consider adding a `--list` or  `--dry-run` option to `docker engine update`.

Reply:
`docker engine upgrades`?